src/main/java/tech/gruppone/stalker/server/repositories/OrganizationRoleRepository.java

```diff
@@ -0,0 +1,14 @@
+package tech.gruppone.stalker.server.repositories;
+
+import org.springframework.data.r2dbc.repository.Query;
+import org.springframework.data.repository.reactive.ReactiveCrudRepository;
+import reactor.core.publisher.Flux;
+import tech.gruppone.stalker.server.model.db.OrganizationRoleDao;
+
+public interface OrganizationRoleRepository
+    extends ReactiveCrudRepository<OrganizationRoleDao, Long> {
+
+  @Query(
+      "SELECT * FROM OrganizationRole r, Organization o, User u, AdministratorType a WHERE r.organizationId = o.id AND r.userId = u.id AND r.administratorType = a.id AND r.organizationId = :id")
+  Flux<OrganizationRoleDao> findUsersRoles(long id);
```

Comment:
findUserRolesByOrganizationId

Reply:
praticamente se usiamo il "mio" metodo l'annotazione `@Query` non serve più

Reply:
Però se togli l'annotazione query, va a finire che prende SOLO i campi della tabella OrganizationRole, quando io invece ho bisogno anche di un altro campo in successione. 
Se provi a vedere se va senza @Query, allora togliamola pure, altrimenti è meglio lasciarlo. In questi casi, la @Query serve